home-assignments/session1/excercise2.py

```diff
@@ -24,6 +12,8 @@
 api_key = 'a5b74bcd1a10b3ee324a0bf25c1b247d'
 
 def get_geo_by_ip():
+    '''check your location according to your IP.
+    Then check the current weather at your location and writes the result to a file in a regular text format.'''
     send_url = "http://api.ipstack.com/109.64.9.109?access_key=c755c7cae07cf8943a551ee0562b3825"
```

Comment:
It was nicer if u could resolve your IP dynamically instead of hard coded...
though i also have some issues with it (: